.identity/03_managed_identity_ci.tf

```diff
@@ -0,0 +1,18 @@
+module "identity_ci" {
+  source = "github.com/pagopa/terraform-azurerm-v3//github_federated_identity?ref=v7.34.0"
+
+  prefix    = var.prefix
+  env_short = var.env_short
+  domain    = var.domain
+
+  identity_role = "ci"
+
+  github_federations = var.ci_github_federations
```

Comment:
If you wish, you can rid of this variable and use something like:

```hcl
github_federations = [
    {
      repository = "io-web-profile-backend"
      subject    = github_repository_environment.prod_ci.environment
    }
  ]
  ```

Reply:
thank you, done in [42a1bd0](https://github.com/pagopa/io-web-profile-backend/pull/32/commits/42a1bd07a2f1b3618634d703ecbf89c979516510)